### DISCUSSION TITLE
Modification of the greenness vegetation fraction

### DISCUSSION THREAD

**Discussion by laurizioA** [NASA-LIS/LISF#1262] [Q&A]
Hello guys!

I created a new climatology for GVF based on GLASS_MODIS products in tiff format, one file for each month.

How do I convert this data to a format that the LIS can read?

Thanks in advance.

**Comment by dmocko** [comment]
Hi @laurizioA 

Please see some existing LISF codes that can read files in tiff format:

https://github.com/NASA-LIS/LISF/blob/master/ldt/params/soils/read_ISRIC_texture.F90

https://github.com/NASA-LIS/LISF/blob/master/lvt/datastreams/GIMMS_MODIS_NDVI/readGIMMSMODIS_NDVIObs.F90

https://github.com/NASA-LIS/LISF/blob/master/lvt/datastreams/USDM/readUSDMobs.F90

As you can see, these use the GDAL library and associated routines.

Please let us know if you are able to get this working or not.
